presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSource.java

```diff
@@ -125,11 +129,12 @@ public OrcPageSource(
 
             hiveColumnIndexes[columnIndex] = column.getHiveColumnIndex();
 
-            if (column.isPartitionKey()) {
+            if (column.isPartitionKey() || column.isHidden()) {
```

Comment:
Unlike `RecordCursor` above, this is the only usage of `isHidden` is this class.